core/services/pg/connection.go

```diff
@@ -60,8 +61,27 @@ func NewConnection(uri string, dialect dialects.DialectName, config ConnectionCo
 		uri = uuid.New().String()
 	}
 
+	// Set sane defaults for every new database connection.
+	// Those can be overridden with Txn options or SET statements in individual connections.
+	// The default values are the same for Txns.
+	connConfig, err := pgx.ParseConfig(uri)
```

Comment:
@samsondav this fails on txdb uuids. Which is the correct thing to do?
1. Skip it with txdb, using an empty `pgx.ConnConfig`  instead.
2. Parse the original uri, before replacing it with a uuid.

Reply:
Just pushed #1... :crossed_fingers:

Reply:
Oh nvm, it's not actually this easy, because `ParseConfig` was the only thing we do with the uri in the first place. These two cases might need to be handled more separately.

Reply:
Updated with a faux scheme prefix `uuid:` :shrug:

Reply:
Eh, that's not enough. Need to fake more...

Reply:
This approach seems to be incompatible with `txdb`:
```
          	            	failed to connect to `host=4dfea6ca-b6bc-43da-a8bd-a163af63b319 user=runner database=`: hostname resolving error (lookup 4dfea6ca-b6bc-43da-a8bd-a163af63b319 on 127.0.0.53:53: server misbehaving) 
```

Reply:
So I think I can make it "work" by only using `pgx.ParseConfig` with the `pgx` dialect, but then will our tests using `txdb` remain in a broken state?